qiita_db/util.py

```diff
@@ -1057,6 +1058,30 @@ def get_timeseries_types():
         return qdb.sql_connection.TRN.execute_fetchindex()
 
 
+def get_pubmed_ids_from_dois(doi_ids):
+    """Get the dict of pubmed ids from a list of doi ids
+
+    Parameters
+    ----------
+    doi_ids : list of str
+        The list of doi ids
+
+    Returns
+    -------
+    dict of {doi: pubmed_id}
+        Return dict of doi and pubmed ids
+
+    Notes
+    -----
+    If doi doesn't exist it will not return that {key: value} pair
+    """
+    with qdb.sql_connection.TRN:
+        sql = "SELECT doi, pubmed_id FROM qiita.publication WHERE doi IN %s"
+        qdb.sql_connection.TRN.add(sql, [tuple(doi_ids)])
+        return {row[0]: join(*row[1:])
```

Comment:
Shouldn't this just be `{row[0]: row[1] for row in qdb.sql_connection.TRN.fetchindex()}`?